### ISSUE TITLE
[UI  and Accessibility Bug] Broken icons on the review screen and missing alt prop

### ISSUE THREAD

**Issue by diegormb06** [topcoder-platform/Training-Challenges#184]
Steps to Reproduce: 
1. Open a challenge overview
2. Click on review scorecard: 
![image](https://user-images.githubusercontent.com/22642060/98110607-17dcf000-1e7e-11eb-828b-127c69a542c3.png)

Actual Result: 
In the "my projects" and "All open projects" tab there are some broken icons

Expected Result: 
See all icons and images with alt prop

Screenshots or screencast URL: 
![image](https://user-images.githubusercontent.com/22642060/98111560-89696e00-1e7f-11eb-917b-eac3f1223227.png)

Device/OS/Browser Information: 
Google Chrome
Version 86.0.4240.183 64 bits
